app/boards/shields/leeloo/Kconfig.shield

```diff
@@ -1,8 +1,21 @@
 # Copyright (c) 2022 The ZMK Contributors
 # SPDX-License-Identifier: MIT
 
+config SHIELD_LEELOO
+    bool
+
```

Comment:
I haven't seen this used elsewhere. I don't have anything specifically against it, but it turns one or operator line into four extra select lines here. Not sure if that's a huge benefit

Reply:
Thank you for our conversation @Nicell. :)

Reply:
I think this pattern is sort of established now for versioned shields, like the Kyria. @Nicell Do you still have this concern? (I am missing the context of any conversation that happened elsewhere.)